React/Views/RCTView.m

```diff
@@ -716,6 +716,21 @@ - (NSString *)description
 #endif // TODO(macOS GH#774)
 
 #if TARGET_OS_OSX // [TODO(macOS GH#774)
+- (NSShadow*)shadow {
+  CALayer *layer = self.layer;
+
+  // Workaround AppKit issue with directly manipulating the view layer's shadow.
```

Comment:
Nit: I think the comment is better of closer to the #IF TARGET_OS_OSX tag, aka put it above line 719